front/components/items/Appointment.tsx

```diff
@@ -53,22 +57,28 @@ const Appointment = () => {
     }
     const allDates: string[] | undefined = appointmentsTherapist?.map(item => item.attributes.date);
```

Comment:
This would be great for you to create type for appointmentsTherapist. In that case you could add it in line 31 so now you wouldn't have to allign type to variable